recipes/devel/compat/binutils.yaml

```diff
@@ -2,6 +2,9 @@ inherit: [autotools]
 
 metaEnvironment:
     PKG_VERSION: "2.34"
+    PKG_LICENSE: |
+        FSFAP AND GFDL-1.3 AND GPL-2.0-or-later AND GPL-3.0-or-later AND
```

Comment:
Same here with `>-`. BTW, isn't `GPL-2.0-or-later AND GPL-3.0-or-later` the same as `GPL-2.0-or-later`. At least as long as the individual lincnes are not attributed to specific multiPackages...